.github/workflows/ci.yml

```diff
@@ -0,0 +1,10 @@
+name: CI
```

Comment:
Seems like we're missing things like auto merge-ups. Did you run module-standardiser? If not could you add as a "do later" AC on the parent issue

Reply:
Running standardiser before there's any code or config files will at best give incomplete results. There's a note under "Still needs to be done" in the issue for this.